src/components/TextField/SbTextField.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="sb-textfield">
+    <label v-if="label" :for="id" class="sb-textfield__label">{{
+      label
+    }}</label>
+    <div class="sb-textfield__inner">
+      <span
+        v-if="prefix"
+        class="sb-textfield__inner__input--with-text-model-left"
```

Comment:
I believe it's more clear use the class `sb-textfield__prefix`. What do you think?